docs/guides/wifi-bluetooth.md

```diff
@@ -9,7 +9,7 @@ Check if this command outputs any lines: `modinfo brcmfmac | grep 4387` If it do
 Refer to the "Updating Kernel" section on your distro's FAQ for instructions if you need to update your kernel:
 
 - [Arch](https://wiki.t2linux.org/distributions/arch/faq/#updating-kernel)
-- [Fedora](https://wiki.t2linux.org/distributions/fedora/faq/#updating-kernel)
+- [Fedora](https://docs.fedoraproject.org/en-US/quick-docs/securing-the-system-by-keeping-it-up-to-date/#manual-updating-using-gui)
```

Comment:
As per mikeeq’s repo instructions, the def repo has been introduced in fedora 37, and I believe that no user of Fedora 37 would have to follow these instructions since all releases of the same have Asahi Linux patches.

If a person is in a situation that he had to follow the “Updating Kernel” section, it means that they are using some old release, and as per mikeeq’s repo, the dnf way shouldn’t work for them right?


So I guess we should keep it as such? The faq changes look fine though.

Reply:
> If a person is in a situation that he had to follow the “Updating Kernel” section, it means that they are using some old release, and as per mikeeq’s repo, the dnf way shouldn’t work for them right?

I didnt think of it that way, I linked it to mikeeqs instructions now.